### PR TITLE
Implement LM pool deployer

### PR DESCRIPTION
**Pull request by kettlebell**

### CODE REVIEW

File: spectrum-offchain-lm/src/data/order.rs [hunk 1]
```diff
@@ -351,6 +390,8 @@ pub struct Redeem {
     pub redeemer_prop: ErgoTree,
     pub bundle_key: TypedAssetAmount<BundleKey>,
     pub expected_lq: TypedAssetAmount<Lq>,
+    pub miner_prop_bytes: Vec<u8>,
```
Comment (oskin1) [inline spectrum-offchain-lm/src/data/order.rs:393]: MinerProposition is constant, so no need to parse it.

**Review by oskin1 — APPROVED** [review]
Looks good. Please see my remarks.

File: spectrum-offchain-lm/src/data/order.rs (hunk 1 above)
Comment (kettlebell) [reply spectrum-offchain-lm/src/data/order.rs:393]: So we just need to check against this: https://github.com/ergoplatform/sigma-rust/blob/46c70f58d5f552b08c20af4fc8af91386e488aae/ergo-lib/src/wallet/miner_fee.rs#L8-L9
?

Comment (oskin1) [reply spectrum-offchain-lm/src/data/order.rs:393]: Yes

Comment (kettlebell) [reply spectrum-offchain-lm/src/data/order.rs]: Thanks, all fixed.

**Pull request merged**